tsl/test/isolation/specs/freeze_chunk.spec.in

```diff
@@ -0,0 +1,93 @@
+# This file and its contents are licensed under the Timescale License.
+# Please see the included NOTICE for copyright information and
+# LICENSE-TIMESCALE for a copy of the license.
+
+#check concurrent operations with freeze_chunk 
+setup {
+  DROP TABLE IF EXISTS measurements;
+  CREATE TABLE measurements (time timestamptz, device int , temp float);
+  SELECT create_hypertable('measurements', 'time', chunk_time_interval=>'7 days'::interval);
+  INSERT INTO measurements VALUES ('2020-01-03 10:30', 1, 1.0), ('2020-01-03 11:30', 2, 2.0);
+  ALTER TABLE measurements SET (timescaledb.compress);
+
+  CREATE OR REPLACE FUNCTION debug_waitpoint_enable(TEXT) RETURNS VOID LANGUAGE C VOLATILE STRICT
+  AS '@TS_MODULE_PATHNAME@', 'ts_debug_point_enable';
+
+  CREATE OR REPLACE FUNCTION debug_waitpoint_release(TEXT) RETURNS VOID LANGUAGE C VOLATILE STRICT
+  AS '@TS_MODULE_PATHNAME@', 'ts_debug_point_release';
+  CREATE OR REPLACE FUNCTION lock_chunktable( name text) RETURNS void AS $$
+    BEGIN EXECUTE format( 'lock table %s IN SHARE MODE', name);
+    END; $$ LANGUAGE plpgsql;
+
+}
+
+teardown {
+  DROP TABLE measurements;
```

Comment:
Don't need to cleanup all objects created in the `setup` ?

Reply:
the objects are recreated by each session. create or replace function takes care of it.